target/hexagon/idef-parser/parser-helpers.h

```diff
@@ -87,14 +90,14 @@ void out_assert(Context *c, YYLTYPE *locp, void *dummy);
 void commit(Context *c);
 
 #define OUT_IMPL(c, locp, x)                    \
-    _Generic(*x,                                \
-        char:      str_print,                   \
-        uint8_t:   uint8_print,                 \
-        uint64_t:  uint64_print,                \
-        int:       int_print,                   \
-        unsigned:  uint_print,                  \
-        HexValue:  rvalue_out,                  \
-        default:   out_assert                   \
+    QEMU_GENERIC(*x,                            \
+        (char,     str_print),                  \
+        (uint8_t,  uint8_print),                \
+        (uint64_t, uint64_print),               \
+        (int,      int_print),                  \
+        (unsigned, uint_print),                 \
+        (HexValue, rvalue_out),                 \
+        out_assert                              \
```

Comment:
Actually, nope, drop this. I've just rebased on the latest QEMU and they have switched to C11:

```
$ git grep -w gnu11 | cat
configure:    CONFIGURE_CXXFLAGS=$(echo "$CONFIGURE_CFLAGS" | sed s/-std=gnu11/-std=gnu++11/)
configure:CONFIGURE_CFLAGS="-std=gnu11 -Wall"
meson.build:        default_options: ['warning_level=1', 'c_std=gnu11', 'cpp_std=gnu++11', 'b_colorout=auto'] +
```

Also, the macro `QEMU_GENERIC` no longer exists!

I'll soon push the branch rebased on today QEMU's `master`

Reply:
Haha alright!